docs/addons/ack-addon.md

```diff
@@ -0,0 +1,115 @@
+# AWS Controller for Kubernetes Add-on
+
+This add-on installs [aws-controller-8s](https://github.com/aws-controllers-k8s/community).
+
+AWS Controllers for Kubernetes (ACK) lets you define and use AWS service resources directly from Kubernetes. With ACK, you can take advantage of AWS managed services for your Kubernetes applications without needing to define resources outside of the cluster or run services that provide supporting capabilities like databases or message queues within the cluster.
+
+ACK is an open source project built with ❤️ by AWS. The project is composed of many source code repositories containing a common runtime, a code generator, common testing tools and Kubernetes custom controllers for individual AWS service APIs.
+
+## Usage
+
+> Pattern # 1 : This installs AWS Controller for Kubernetes for IAM ACK Controller. This uses all default parameters for installation of the IAM Controller.
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.AckAddOn({

```

Comment:
Please drop skipVersionValidation - the fix is merged now, it will skip OCI. Please let me know if you see any issues.

Reply:
Done